src/shared/ui/survey/tests/TextParagraph.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import renderer from 'react-test-renderer';
+
+import TamaguiProvider from '@app/app/ui/AppProvider/TamaguiProvider';
+import { LongTextInput } from '@shared/ui';
+
+import TextParagraph from '../TextParagraph';
+
+jest.mock('react-i18next', () => ({
+  useTranslation: jest.fn().mockImplementation(() => ({
+    t: jest.fn().mockImplementation(key => key),
+  })),
+}));
+
+describe('TextParagraph Component Tests', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders correctly with expected props', () => {
```

Comment:
Please consider naming pattern like in other tests: "Should [expected result] when [conditions]"